radareorg/radare2
# Decouple shlr/gdb registers profile from code

**Your checklist for this pull request**

**Detailed description**

* Sync r2 debugger registers profile with `shlr/gdb`
* Add new `set_reg_profile` function to r_debug plugin
* Add `set_reg_profile` to `r_debug_plugin_gdb`
* Change `drp` to sync r_debug plugin register profile with r2 debugger

**Test plan**
* Connect r2 to a remote gdbserver and run `drp regprofile` with a different registers profile.
* Change registers value with `dr` and verify changes.
* Set registers value with `dr register=newValue`  and verify changes. 

Test is not written for this pull request yet because I could not figure out a way to setup a gdbserver and run the test on CI.

**Closing issues**
closes #15928
